### ISSUE TITLE
README needs updating

### ISSUE THREAD

**Issue by LeiRui** [thulab/iotdb-benchmark#82]
'config.properties':
- `CACHE_NUM` is now changed to `BATCH_SIZE`, I guess
- `IS_GEN_DATA` doesn't exist now
- The usage of `BENCHMARK_WORK_MODE` is not mentioned in the Readme. For example, if I want to use the "Sample Data Generation" feature, I should use model `insertTestWithUserDefinedPath`. But this point is missing in the readme.

**Comment by LeiRui** [comment]
Plus, in the `Sample Code Generation` chapter, the example routine:
```
IS_GEN_DATA=true
STORAGE_GROUP_NAME=root.ln
TIMESERIES_NAME=wf01.wt01.status
TIMESERIES_TYPE=BOOLEAN
ENCODING=PLAIN
TIMESERIES_VALUE_SCOPE=true,false
TEST
TIMESERIES_NAME=wf01.wt01.temperature
TIMESERIES_TYPE=INT32
ENCODING=RLE
TIMESERIES_VALUE_SCOPE=13,16
TEST
TIMESERIES_NAME=wf01.wt01.time
TIMESERIES_TYPE=FLOAT
ENCODING=RLE
TIMESERIES_VALUE_SCOPE=5600.0,6200.0
TEST
TIMESERIES_NAME=wf02.wt02.hardware
TIMESERIES_TYPE=TEXT
ENCODING=PLAIN
TIMESERIES_VALUE_SCOPE=v1,v2
TEST
```
contains timeseries `root.ln.wf01.wt01.time`. However, now in IoTDB, `time` is a reserved keyword, so this timeseries cannot be created. Maybe update this example too.

**Comment by liuruiyiyang** [comment]
Thanks for your notation. We will fix it in the near future :)

**Comment by liuruiyiyang** [comment]
Fixed in PR#84
